CHANGES.md

```diff
@@ -5,8 +5,15 @@
 ### Features
 
 * Support treetime 0.11.* [#1310][] (@corneliusroemer)
+* Allow minimal export using only a (newick) tree in `augur export v2`. [#1299][] (@jameshadfield)
+* A number of schema updates and improvements [#1299][] (@jameshadfield)
+    * We now require all nodes to have `node_attrs` on them with one of `div` or `num_date` present
+    * Some never-used properties are removed from the schemas, including a pattern for defining nucleotide INDELs which was never used by augur or auspice.
+    * Tip label defaults are now settable within the auspice-config JSON
```

Comment:
Allowed per validation in auspice-config but not yet supported by current auspice 🙃

Reply:
Yeah totally, I forgot that auspice PR hadn't been merged + released. I'll do that today

Reply:
[Auspice 2.49.0 released](https://github.com/nextstrain/auspice/releases/tag/v2.49.0) which parses this display_default. It'll appear on nextstrain.org / auspice.us etc over the coming day or two